### DISCUSSION TITLE
Adding Brian Cash as Triage member

### DISCUSSION THREAD

**Discussion by kcpevey** [nebari-dev/nebari#2567] [Community]
Brian Cash has taken up [an issue](https://github.com/nebari-dev/nebari/issues/2451) and [opened a PR](https://github.com/nebari-dev/nebari/pull/2561) for the issue.

I propose that he be added to the traige team so that he can be assigned to the issue and PR. 

gh username:  `BrianCashProf`

**Comment by dharhas** [comment]
Invitation sent. cc: @BrianCashProf
